src/index.html

```diff
@@ -5,8 +5,520 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo-moyo">
+        <img
+          src="images/logo.png"
+          alt="images"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--is-active">
+            <a
+              href="#"
+              class="nav__link nav__link--is-active is-active"
+            >
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
            >
```
Клас nav__link--is-active не потрібен. Стилізуй просто is-active.
До речі, is-active треба вішати саме на тег `<a>`, а не на `<li>`